components/hab/install.sh

```diff
@@ -182,12 +191,27 @@ get_version() {
   fi
 }
 
+get_target() {
+  if [ -z "${target}" ]; then
+    target="${arch}-${sys}"
+  else
+    if [ "${sys}" != "linux" ]; then 
+      _e="--target is only valid on Linux systems"
+      exit_with "$_e" 7
+    elif [ "${target%-kernel2}" != "x86_64-linux" ]; then
```

Comment:
I wonder if it might be a little easier to read (and more future proof, as we may add additional targets) to structure like:
```bash
  if [ -z "${target}" ]; then
    target="${arch}-${sys}"
  elif [ "${sys}" != "linux" ]; then 
    …
  else
    case $target in
      x86_64-linux | x86_64-linux-kernel2)
        ;;
      *)
        _e="${target} is not a valid target. Please specify one of: "
        _e="${_e} ['x86_64-linux', 'x86_64-linux-kernel2']"
        ;;
    esac
  fi
```